presto-main/src/main/java/com/facebook/presto/metadata/PolymorphicScalarFunction.java

```diff
@@ -115,19 +119,61 @@ public ScalarFunctionImplementation specialize(BoundVariables boundVariables, in
         List<Object> extraParameters = computeExtraParameters(matchingMethodsGroup.get(), context);
         MethodHandle matchingMethodHandle = applyExtraParameters(matchingMethod.get(), extraParameters);
 
-        return new ScalarFunctionImplementation(nullableResult, nullableArguments, matchingMethodHandle, deterministic);
+        return new ScalarFunctionImplementation(nullableResult, nullableArguments, getIsNullAnnotations(matchingMethod.get()),
```

Comment:
When arguments do not fit on one line, format it so that each argument occupies an entire line.